cli/start.go

```diff
@@ -17,7 +19,22 @@ var (
 )
 
 func startRun(cmd *cobra.Command, args []string) {
-	req, err := createRequest(args)
+	group := ""
+	switch len(args) {
+	case 1:
+		group = args[0]
+	case 0:
+		fallthrough
+	default:
+		cmd.Help()
+		os.Exit(1)
+	}
+
+	newRequestConfig := controller.DefaultRequestConfig()
```

Comment:
What is a operation in your sense? What is complex? 

We want to do the start. This is done using the controller. The controller needs a request. 

To create the request we create the request instance using its configuration. 

My goal was to have a well defined API.